lib/http/response/body.rb

```diff
@@ -10,6 +10,8 @@ class Body
       include Enumerable
       def_delegator :to_s, :empty?
 
+      attr_reader :client
```

Comment:
So, it appears as if the `@client` instance variable is in fact an `HTTP::Connection`. In that case, I suggest renaming it (and the `initialize` param) to `@connection` and exposing an `attr_reader: connection` here.